go/pkg/fakes/server.go

```diff
@@ -74,6 +74,7 @@ func (s *Server) NewTestClient(ctx context.Context) (*rc.Client, error) {
 // TestEnv is a wrapper for convenient integration tests of remote execution.
 type TestEnv struct {
 	Client     *rexec.Client
+	GrpcClient *rc.Client
```

Comment:
Why do we need the grpc client here? Can't we access it from the rexec.Client?

Reply:
Oh, right! I forgot it was public there! Sure, I can use it from there. I thought to expose it for the intermediate layer tests, which don't care about rexec.